### DISCUSSION TITLE
My rights suddenly expired after launch. Why is that?

### DISCUSSION THREAD

**Discussion by BigLiuH** [community/community#147861] [GitHub Education]
### Hi everyone,

My rights suddenly expired after launch. Why is that?

**Comment by davevad93** [comment]
Hi @BigLiuH ,

There must have been an issue with your application, contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem.

**Comment by samus-aran** [comment]
@BigLiuH, you need to give us more information.

>  My rights suddenly expired after launch. Why is that?

Where are reading this from? Verification usually only needs to be renewed every so often, certainly not right after unless there was a problem on our end. From what I can see your application has been successful and you should have access to your academic benefits 👍
